.github/workflows/kind.yml

```diff
@@ -83,7 +83,6 @@ jobs:
 
     - name: Verify Deployment Configuration
       run: |
-        make build-images
```

Comment:
This is explicitly being called in the framework tests also, so either this needs to stay here and the `build-images` removed from the Make target in this repo OR it needs to be removed from the framework workflow:
https://github.com/stolostron/governance-policy-framework/blob/0903452ed5de2b5d99eba286f4a16ef1007ed064/.github/workflows/kind.yml#L159-L161

Reply:
This PR goes with the former strategy: https://github.com/stolostron/config-policy-controller/pull/531#pullrequestreview-1507628578

Reply:
Okay, I'll remove it from the Make target instead.